app/src/main/java/org/oppia/app/topic/info/TopicInfoFragment.kt

```diff
@@ -12,6 +12,18 @@ import javax.inject.Inject
 
 /** Fragment that contains info of Topic. */
 class TopicInfoFragment : InjectableFragment() {
+  companion object{
+    /** Returns a new [TopicInfoFragment]. */
+    fun newInstance(internalProfileId: Int, topicId: String): TopicInfoFragment{
+      val topicInfoFragment = TopicInfoFragment()
+      val args = Bundle()
+      args.putInt(PROFILE_ID_ARGUMENT_KEY,internalProfileId)
```

Comment:
Nit: Space after `,` 
`args.putInt(PROFILE_ID_ARGUMENT_KEY, internalProfileId)`

Reply:
Are we not including fragment name in key? @rt4914 Can we decide the naming  convention that we may follow in different situations also?

Reply:
Key names convention is a separate issue https://github.com/oppia/oppia-android/issues/747

Reply:
Done (Fixed the space issue)

Reply:
Looks good. Thanks.